test/views/pages/profile/profile_page_test.dart

```diff
@@ -265,5 +264,33 @@ void main() {
       final profilePage = find.byType(ProfilePage);
       expect(profilePage, findsOneWidget);
     });
+
+    testWidgets("Check that user centauri points count updates correctly.",
+        (tester) async {
+      await tester.pumpWidget(profileProviderScope(fakeFireStore, homePageApp));
+      await tester.pumpAndSettle();
+
+      final userPoints = expectedUser.data.centauriPoints.toString();
+
+      // Navigate to profile
+      await tester.tap(find.byKey(AppTopBar.profilePictureKey));
+      await tester.pumpAndSettle(delayNeededForAsyncFunctionExecution);
+
+      // Check correct centauri points
+      expect(find.textContaining(userPoints), findsOneWidget);
+
+      // Navigate back to home page
+      await tester.tap(find.byType(LeadingBackButton));
+      await tester.pumpAndSettle();
+
+      // Change the user points
+      await userCollection
+          .doc(expectedUser.uid.value)
+          .set(expectedUser.data.withPointsAddition(10).toDbData());
+
+      // Navigate back to profile page
+      await tester.tap(find.byKey(AppTopBar.profilePictureKey));
+      await tester.pumpAndSettle(delayNeededForAsyncFunctionExecution);
+    });
```

Comment:
```suggestion

      // Check updated centauri points
      final updatedPoints = expectedUser.data.centauriPoints.toString();
      expect(find.textContaining(updatedPoints), findsOneWidget);
    });
```

I think you forgot to check the updated points.

Reply:
Right, mb thank you. It was in my local branch, but somehow I forgot to stage the changes...

I committed it in 1d59c7a. Please mark as resolve when it's ok for you.